indico/modules/events/surveys/controllers/display.py

```diff
@@ -50,7 +53,7 @@ def view_class(self):
 class RHSurveyList(RHSurveyBaseDisplay):
     def _process(self):
         surveys = (Survey.query.with_parent(self.event_new)
-                   .filter(Survey.is_visible)
+                   .filter(Survey.is_visible, ~Survey.private)
```

Comment:
I don't know, do we? The user can still access the incomplete submission through the emailed link, but maybe would be appropriate to show the unfinished submission in the survey list.